app/jobs/content_event_job.rb

```diff
@@ -0,0 +1,39 @@
+# A generic job for sending events about a generic files to a user and their followers.
+#
+# @attr [String] generic_file_id  the id of the file the event is specified for
+#
+class ContentEventJob < EventJob
+  attr_accessor :generic_file_id
+
+  def initialize(generic_file_id, depositor_id)
+    super(depositor_id)
+    self.generic_file_id = generic_file_id
```

Comment:
This isn't a change you've made, but I wonder if this should be `@generic_file_id` instead of `self.generic_file_id` to be more idiomatic.